packages/icons/src/utils/css-helpers.ts

```diff
@@ -0,0 +1,55 @@
+import { rem } from '@littlespoon/theme/lib/utils'
+
+/**
+ * Returns rotation value for given direction.
+ * Works only if icon initially points down
+ */
+export function getRotate(direction: string | undefined): number {
```

Comment:
Would `direction` ever be undefined? Same goes for other params below.

Reply:
We could also explicitly type the values here like:

```ts
direction: 'up' | 'down' | 'left' | 'right'
```

Reply:
It has nullable type in props, for example in ArrowIcon. So I would keep `| undefined`. It defaults to 'down' anyway.

Reply:
It defualts to down on the component level, but it is nullable in the props, so I would keep `| undefined`

Reply:
Without this undefined it shows the error when you pass props.direction. So I would keep it if it's not a must-have. Because it will require a bigger refactor.